nixos/modules/services/networking/openvpn.nix

```diff
@@ -195,9 +385,39 @@ in
               };
             });
           };
+
+          netns = mkOption {
+            default = true;
```

Comment:
I've been testing this PR with and without network namespaces. Works great.
One comment, this should probably default to `null` instead of `true`.

Reply:
@pborzenkov good catch, thanks!
- [X] Fix invalid `netns.default`.
- [X] Fix [incorrect](https://github.com/NixOS/nixpkgs/pull/187905) option aliasing by using `mkAliasAndWrapDefsWithPriority` instead of `mkAliasDefinitions`.